presto-testing/src/main/java/io/prestosql/testing/datatype/CreateAsSelectDataSetup.java

```diff
@@ -40,17 +39,20 @@ public CreateAsSelectDataSetup(SqlExecutor sqlExecutor, String tableNamePrefix)
     public TestTable setupTestTable(List<ColumnSetup> inputs)
     {
         List<String> columnValues = inputs.stream()
-                .map(this::literalInExplicitCast)
+                .map(this::format)
                 .collect(toList());
         Stream<String> columnValuesWithNames = range(0, columnValues.size())
-                .mapToObj(i -> format("%s col_%d", columnValues.get(i), i));
+                .mapToObj(i -> String.format("%s col_%d", columnValues.get(i), i));
         String selectBody = Joiner.on(",\n").join(columnValuesWithNames.iterator());
         return new TestTable(sqlExecutor, tableNamePrefix, "AS SELECT " + selectBody);
     }
 
-    private String literalInExplicitCast(ColumnSetup input)
+    private String format(ColumnSetup input)
```

Comment:
The previous method name was more clear about what it's doing. This one is more generic and also conflicts with String.format.

If you do want to rename I'd prefer something like `formatAsLiteral`.

Reply:
I renamed because the old name is no longer applicable after the method became more flexible.